integration/terraform/ec2/linux/README.md

```diff
@@ -177,16 +176,25 @@ configuring GitHub Actions secrets.
 ## Local setup (Not recommended)
 
 ### Install terraform
+
 Install `terraform` on your local machine ([download](https://www.terraform.io/downloads)).
 
 ### Build and upload agent artifacts
+
 1. Run `make release` to test, build, and generate agent artifacts that can be installed and tested.
-   1. If targeting a specific OS, you can run a more specific make command. `make build && make package-deb` would build and package for Ubuntu.
-2. Copy the artifacts to the test S3 bucket: `aws s3 cp ./build/bin s3://{your bucket name}/integration-test/binary/{commit SHA} --recursive`
-   1. Substitute out the values wrapped in `{}` with what you have for testing
+    1. If targeting a specific OS, you can run a more specific make command. `make build && make package-deb` would
+       build and package for Ubuntu.
+2. Sign the files with your GPG key
```

Comment:
Signing is optional for testing. I only added signing for when we can do release through gh actions.

Reply:
Noted. I'll update that